spec/services/verifier_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'spec_helper'
+
+describe Verifier do
+  let(:parameters) do
+    {
+      :time => 1426064293.0,
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Separate every 3 digits in the integer portion of a number with underscores(_).